packages/static-build/src/index.ts

```diff
@@ -394,6 +395,20 @@ export const build: BuildV2 = async ({
         process.env[key] = value;
       }
 
+      const speedInsightsVersion = getInstalledPackageVersion(
+        '@vercel/speed-insights'
+      );
+
+      const isSpeedInsightsInstalled = Boolean(speedInsightsVersion);
+
+      if (
+        isSpeedInsightsInstalled &&
+        process.env.VERCEL_ANALYTICS_ID &&
+        ['next', 'nuxtjs'].includes(framework.slug || '')
```

Comment:
**though:** shouldn't we include `gatsby` as well?
**suggestion:** maybe we could include debug logs here as in the next.js builder.

Reply:
Oh you're right. I forgot to push that.
+1 on the debug log. Adding it